docs/develop/develop-overview.md

```diff
@@ -0,0 +1,189 @@
+---
+title: Develop Office Add-ins
+description: An introduction to developing Office Add-ins.
+ms.date: 10/28/2019
+localization_priority: Priority
+---
+
+# Develop Office Add-ins
+
+Office Add-ins can extend the UI and functionality of Office applications and interact with content in Office documents. You can use familiar web technologies to create Office Add-ins that extend and interact with Word, Excel, PowerPoint, OneNote, Project, or Outlook, and your solution can run in Office across multiple platforms, including Windows, Mac, iPad, and in a browser. This article provides an introduction to developing Office Add-ins.
```

Comment:
I think by combining the list of hosts and platforms, it implies a single add-in can work on both Word and Excel (just like they work on web and desktop). Consider breaking that into two sentences.